examples/worklet.rs

```diff
@@ -141,15 +143,12 @@ impl AudioProcessor for WhiteNoiseProcessor {
         true // source node will always be active
     }
 
-    fn onmessage(&mut self, msg: Box<dyn std::any::Any + Send + 'static>) {
+    fn onmessage(&mut self, msg: &mut Box<dyn Any + Send + 'static>) {
```

Comment:
```suggestion
    fn onmessage(&mut self, msg: &mut dyn Any) {
```

This will work as well and looks a lot cleaner. Is there any drawback?

Reply:
That didn't work and I don't know why yet.

Reply:
I have dropped the non-working commit. Downcasting doesn't seem to work then. We probably need to keep it boxed for preserving the `Send` and `'static` bounds.

Reply:
Fixed. Working with `Any` is tricky. I simply forgot to invoke `Box::as_mut` instead of using `&mut` when passing the parameter :see_no_evil: